components/aws/aws.app.js

```diff
@@ -1,27 +1,202 @@
+const awsRegions = require("./regions");
```

Comment:
I believe [this API endpoint](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/EC2.html#describeRegions-property) lists the regions enabled for your account (it's under the EC2 class, but it appears like it's not unique to EC2). Can we reference that so we don't have to keep the list static as new regions are added?

Reply:
Done. I'm keeping both options since calling this API requires additional permissions that the account might not have.
What would be the best place to document these permissions? (we don't have a `components/aws/README.md` file yet).

Reply:
You make a good point re: the IAM permissions necessary to make this call. I agree that's hard to communicate given the current state. Sorry to go back and forth on this, but I think we should keep the static list you had until we address that.

Reply:
Ignore the last comment. I looked at the code and I see what you did re: the fallback. Works great.